### PR TITLE
Useless Visual Tweak(s)

### PR DESCRIPTION
**Pull request by MC42**
Capitalized "Home" in "Home Championship."

### CODE REVIEW

**Comment by fangeugene** [comment]
While you're at it, perhaps it's worth capitalizing the rest of the items such as "First competed in XXXX" or "From San Jose, California, USA"

**Comment by MC42** [comment]
I'll look into it @fangeugene momentarily.

File: templates_jinja2/team_partials/team_info.html [hunk 1]
```diff
@@ -13,10 +13,10 @@
   <div class="col-sm-7">
     <p>
       {% if team.location %}<span class="glyphicon glyphicon-map-marker"></span> from <a href="http://maps.google.com/maps?q={{ team.location|urlencode }}" target="_blank">{{ team.location }}</a><br>{% endif %}
-      {% if team.name %}<span class="glyphicon glyphicon-info-sign"></span> aka <em>{{ team.name }}</em><br>{% endif %}
-      {% if team.rookie_year %}<span class="glyphicon glyphicon-calendar"></span> first competed in {{ team.rookie_year }}<br>{% endif %}
+      {% if team.name %}<span class="glyphicon glyphicon-info-sign"></span> a.k.a. <em>{{ team.name }}</em><br>{% endif %}
```
Comment (nwalters512) [inline templates_jinja2/team_partials/team_info.html]: I'm personally a fan of keeping it as "aka"

Comment (phil-lopreiato) [inline templates_jinja2/team_partials/team_info.html]: Ditto

**Comment by MC42** [comment]
I'm pretty much done with minor typographical visual changes for now.

**Comment by fangeugene** [comment]
Looks good!

**Pull request merged**